good_first_issues/graphql/services.py

```diff
@@ -142,6 +162,16 @@ def identify_mode(name: str, repo: str, user: bool) -> Tuple[str, Dict, str]:
         variables["name"] = name
         mode = "org"
 
+    if hacktoberfest and not repo:
+        query = search_query
+        variables["queryString"] = 'topic:hacktoberfest '
+        if name:
+            variables["queryString"]+=f'user:{name}'
+        mode="search"
+
+    if repo and hacktoberfest:
+        spinner.fail("--hacktoberfest or --hf cannot be used with --repo flag")
```

Comment:
The error message is color coded red using `console.print("Error: error message", style="bold red")`

add `sys.exit()` to stop the execution of program for failure state. The program execution is not halted here, and another exception is caught. 

![image](https://user-images.githubusercontent.com/13623913/95541596-78aa1180-0a13-11eb-91d7-b6a187b9c1c3.png)